index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+var semver = require('semver');
+var execa = require('execa');
+var del = require('del');
+
+var exec = function (cmd, args) {
+	const result = execa.sync(cmd, args);
+
+	if (result.stderr !== '') {
```

Comment:
It's probably passed somewhere down the chain. If I pass in `stdio: 'inherit'` I get the error `Cannot read property 'length' of null` when running `execa.sync('npm', ['install'], {stdio: 'inherit'});`